src/api/config/logger-params.md

```diff
@@ -0,0 +1,56 @@
+# Logger Parameters
+
+TODO Explain module
+
+## `logger.level`
+
+- **ENV:** `LOG`, `LOG_LEVEL`, `IROHA_LOG_LEVEL`
+- **Type:** String
+- **Possible Values:** `"TRACE"`, `"DEBUG"`, `"INFO"`, `"WARN"`, `"ERROR"`,
```

Comment:
will you explain what each of the value mean?

Reply:
Yup, will do (or put a todo, at least)